build/azure-pipelines/package-common-create.yml

```diff
@@ -0,0 +1,57 @@
+parameters:
+- name: OS
+  type: string
+  default:
+- name: Type
+  type: string
+
+steps:
+- task: DownloadPipelineArtifact@2
+  inputs:
+    source: 'specific'
+    project: 'ae14e11c-7eb2-46af-b588-471e6116d635'
```

Comment:
The help says either:

Project | (Required) The project name or GUID from which to download the pipeline artifacts.

I am open either way.  I think this can be the project name.  Let me think on this.